Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "AWSS3"
 uuid = "1c724243-ef5b-51ab-93f4-b0a88ac62a95"
-version = "0.9.8"
+version = "0.9.9"
```

Comment:
I'm somewhat unsure if that's the case, what behaviour could someone depend on? Theoretically you already *need* `list_objects` permissions to do an `isdir` on a non root bucket. I guess this could potentially be an issue for someone that is only doing an `isdir` on a root bucket somehow? I'm fine to bump the minor if you still think that should be the case

Reply:
Bumped the version regardless